lightpath/ui/gui.py

```diff
@@ -44,10 +48,14 @@ class LightApp(Display):
 
     parent : optional
     """
-    shown_types = [dtypes.Attenuator, dtypes.GateValve, dtypes.IPM,
-                   dtypes.LODCM, dtypes.OffsetMirror, dtypes.PIM, PPSStopper,
-                   dtypes.PulsePicker, dtypes.Slits, dtypes.Stopper,
-                   dtypes.XFLS]
+    shown_types = {'Attenuator': AttBase, 'Gate Valve': dtypes.GateValve,
+                   'IPM': IPMMotion, 'LODCM': dtypes.LODCM,
+                   'KBOMirror': KBOMirror, 'OffsetMirror': XOffsetMirror,
+                   'PIM': dtypes.PIM, 'PPSStopper': PPSStopper,
+                   'PulsePicker': dtypes.PulsePicker, 'Slit': dtypes.Slits,
+                   'Stopper': dtypes.Stopper, 'XFLS': dtypes.XFLS,
+                   'SIM_IPIMB': IPIMB, 'SIM_Stopper': Stopper,
+                   'SIM_Crystal': Crystal}
```

Comment:
Do we have to consider the placeholder items here for the devices that didn't load properly? Or is that baked in already?

Reply:
Ah, I didn't include those.  I can do that.

Reply:
Actually, I didn't forget it before (I just forgot to write down why I made that decision).  Because all the sim devices inherit from `Valve`, unchecking it removes all the sim devices.  While in sim-mode you can just put the devices back, in normal mode we mock devices with either `Valve` or `Crystal`, the latter of the two inheriting from `Valve`.  🤔

Reply:
But surely there should be a way in non-sim mode to hide the mock devices? Either with their own button or with the original button for that class?

Reply:
Agreed, I just need to change the mock classes so Crystal isn't also Valve